config/menus/maps.cfg

```diff
@@ -1,210 +1,88 @@
-reset_map_ui = 0
-selected_map = 0
-selected_mutators = 0
-selected_mode = -1
-last_mode = -1
-last_mutators = 0
+// CC BY-SA 4-0 https://creativecommons.org/licenses/by-sa/4.0/legalcode
+// Iceflower iceflower@gmx.de
+// Modified by MoonPadUSer moonpaduser@mailbox.org
+
+
+truncate_string = [
+    if (!=s (limitstring $arg1 $arg2) $arg1) [
+        result (concatword (limitstring $arg1 (- $arg2 3)) "...")
+    ] [
+        result (limitstring $arg1 $arg2)
+    ]
+]
+
+if (getalias hidemappreviews) [] [ hidemappreviews = 0 ]
+setpersist hidemappreviews 1
+setcomplete hidemappreviews 1
+
+chosenmode = -1
+chosenmuts = 0
+chosenmap = ""
+chosendemo = -1
+chosendemoimage = ""
+chosendemofilename = ""
 
-map_index = 0
-last_mode = -1 // already created previously
-last_mutators = 0 // already created previously
-map_number = -1
-current_map = ""
-map_pth = ""
-maplist = ""
-map_path = ""
-map_extra = ""
-map_image = ""
-map_octa = 0
 if (getalias mapfavs) [] [mapfavs = ""]
 setpersist mapfavs 1
 setcomplete mapfavs 1
 if (getalias demofavs) [] [demofavs = ""]
 setpersist demofavs 1
 setcomplete demofavs 1
```

Comment:
Again with these, you do not need the extra code to check if they exist; config.cfg will override them anyway.

Reply:
yep, will do